tools/compare_ast.py

```diff
@@ -44,11 +44,11 @@ def compare_ast(cppcheck_parameters: str):
         print('saving cppcheck ast in file ast1 and clang ast in file ast2')
 
         f = open('ast1', 'wt')
-        f.write(ast1);
```

Comment:
oops!! I have fixed this upstream.

Reply:
Yes I saw this. I have now merged your fix.